### DISCUSSION TITLE
callback functions are not working of events object

### DISCUSSION THREAD

**Discussion by hemant-vegam** [dmuy/MDTimePicker#26] [Q&A]
i was trying to use jQuery position method to display the timepicker near the input but callback fuctions are not working even timeChanged also not working

how can we check?

**Comment by dmuy** [comment]
Hi @hemant-vegam 

Sorry for the very late response, what do you mean by "position the timepicker near the input"? There's a configuration for that to make it an inline display instead of a modal.

If you could maybe make a fiddle or demo of your problem, I would appreciate it.

**Reply by hemant-vegam** [reply]
Hi, May be in my code some mistakes because on jsfiddle all events are working fine.  
i can't see any configuration and demo for inline display time picker: [](https://github.com/dmuy/MDTimePicker/wiki/Options#configurations)
can you please share the link for inline display time picker?

Thank for you response👍

**Reply by dmuy** [reply]
@hemant-vegam So sorry, I confused this with the datepicker. Currently there's no inline option. I might add this in the future updates. Thanks
